python/triton/experimental/gluon/language/amd/cdna3/__init__.py

```diff
@@ -97,3 +97,54 @@ def mfma(a, b, acc, _semantic: GluonSemantic = None):
     handle = _semantic.dot(a, b, acc, input_precision=knobs.language.fp32_default, max_num_imprecise_acc=None,
                            out_dtype=acc.dtype).handle
     return ttgl.tensor(handle, ret_type)
+
+
+op_str_to_op = {
+    "max": ir.ATOMIC_OP.MAX, "min": ir.ATOMIC_OP.MIN, "fadd": ir.ATOMIC_OP.FADD, "add": ir.ATOMIC_OP.ADD, "and":
+    ir.ATOMIC_OP.AND, "or": ir.ATOMIC_OP.OR, "xor": ir.ATOMIC_OP.XOR, "xchg": ir.ATOMIC_OP.XCHG
+}
+
+
+@builtin
+def buffer_atomic_rmw(op, ptr, offsets, value, mask=None, sem=None, scope=None, stride=None, _semantic=None):
+    """
+    AMD Buffer atomic RMW operation. Buffer atomics are similar to normal atomics, but access global memory via a
+    scalar base pointer and a tensor of offsets instead of a tensor of pointers.
+    Similar to other buffer ops, the `mask` is a boolean vector that determines if a given element should be processed with
+    the atomic RMW op. Elements with `mask[i] == 0` are dropped (i.e., the atomic is not executed).
+    Similar to TT_AtomicRMWOp: Buffer atomic RMW ops load data at $ptr, do $rmw_op with $val, and store result to $ptr with
```

Comment:
Like I commented earlier for `buffer_load` and `buffer_store`, let's revise the comment to be more kernel developer facing--`TT_AtomicRMWOp` and `$ptr` is referencing to compiler internals..